packages/core/scripts/hardhat/tasks/finder.js

```diff
@@ -0,0 +1,45 @@
+const { task } = require("hardhat/config");
+const { waitForTxn } = require("./utils");
+const { stringToBytes32, interfaceName } = require("@uma/common");
+
+task("setup-finder", "Points Finder to DVM system contracts")
+  .addFlag("registry", "Use if you want to set Registry")
```

Comment:
this is cool. will be very powerful for us to define all logic and scripts using these lags.

Reply:
Agreed, but see Matt's comment about coupling ourselves to another framework. I tend to agree with him, but maybe we should (1) make sure all current scripts are compatible with `yarn hardhat run` and (2) rather than replace all scripts with tasks, add some commonly used tasks that we might want to run via CLI!